### PR TITLE
LRQA-53955 | Master

### PR DESCRIPTION
**Pull request by nkkwon**
https://issues.liferay.com/browse/LRQA-53955
AppBuilder#DeletePublishing

### CODE REVIEW

**Comment by nkkwon** [comment]
ci:test:sf

**Comment by nkkwon** [comment]
ci:test:relevant

**Comment by alexcandidos** [comment]
ci:test:app-builder

File: modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase [hunk 1]
```diff
@@ -1225,6 +1225,40 @@ definition {
 		AppBuilderAdmin.deleteEntry(itemName = "My Form View 1");
 	}
 
+	@description = "Delete a publishing"
+	@priority = "5"
+	test DeletePublishing {
+		property portal.acceptance = "false";
```
Comment (alexcandidos) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase]: hey @nkkwon , the test is okay! could you please just remove this annotation related to acceptance test? Thanks!

**Comment by alexcandidos** [comment]
ci:test:app-builder

**Comment by alexcandidos** [comment]
Pull request forwarded to https://github.com/brianchandotcom/liferay-portal/pull/84085. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**